test/bridge/testpyquante.py

```diff
@@ -10,19 +10,64 @@
 import numpy
 
 from cclib.bridge import cclib2pyquante
+from ..test_data import getdatafile
+from cclib.parser.utils import find_package
+
+from numpy.testing import assert_array_almost_equal
 
 
 class PyquanteTest(unittest.TestCase):
     """Tests for the cclib2pyquante bridge in cclib."""
 
+    def setUp(self):
+        super(PyquanteTest, self).setUp()
+        self._found_pyquante = find_package("PyQuante")
+        self.data, self.logfile = getdatafile("Gaussian", "basicGaussian16", ["water_ccsd.log"])
+
     def test_makepyquante(self):
+        # Test older PyQuante bridge
         from PyQuante.hartree_fock import hf
-        atomnos = numpy.array([1, 8, 1],"i")
-        a = numpy.array([[-1, 1, 0], [0, 0, 0], [1, 1, 0]], "f")
-        pyqmol = cclib2pyquante.makepyquante(a, atomnos)
+        from PyQuante.Molecule import Molecule
+
+        reference = Molecule(
+            "h2o",
+            [(8, (0, 0, 0.119159)), (1, (0, 0.790649, -0.476637)), (1, (0, -0.790649, -0.476637)),],
+            units="Angstroms",
+        )
+        refen, reforbe, reforbs = hf(reference)
+
+        pyqmol = cclib2pyquante.makepyquante(self.data)
         en, orbe, orbs = hf(pyqmol)
-        ref = -75.824754
-        assert abs(en - ref) < 1.0e-6
+
+        self.assertAlmostEqual(en, refen, delta=1.0e-6)
+
+
+class pyquante2Test(unittest.TestCase):
+    """Tests for the cclib2pyquante bridge in cclib."""
+
+    def setUp(self):
+        super(pyquante2Test, self).setUp()
+        self._found_pyquante2 = find_package("pyquante2")
+        self.data, self.logfile = getdatafile("Gaussian", "basicGaussian16", ["water_ccsd.log"])
+
+    def test_makepyquante(self):
+        # Test pyquante2 bridge
+        from pyquante2 import molecule, rhf, h2o, basisset
+
+        bfs = basisset(h2o)
+        # Copied from water_ccsd.log
```

Comment:
Is it possible to read in the file and grab the data instead?

Reply:
From what I know, `pyquante2` only has xyz reader as a means of directly reading in files. I suppose that it is possible to read in from xyz files in turbomole 7.2 data files, although I am not sure if this is what we want given that parsing turbomole is still under development. (`atomcoords` and `atomnos` are already fully implemented for turbomole parser. `atomcharges` isn't, but it shouldn't be an issue here.)

Reply:
I'm going to mark this resolved